crates/typst-library/src/compute/calc.rs

```diff
@@ -356,75 +356,66 @@ pub fn atan2(
     Angle::rad(f64::atan2(y.float(), x.float()))
 }
 
-/// Calculates the hyperbolic sine of an angle.
-///
-/// When called with an integer or a float, they will be interpreted as radians.
+/// Calculates the hyperbolic sine of a hyperbolic angle.
 ///
 /// ## Example { #example }
 /// ```example
 /// #calc.sinh(0) \
-/// #calc.sinh(45deg)
+/// #calc.sinh(1.5)
 /// ```
 ///
 /// Display: Hyperbolic sine
 /// Category: calculate
 #[func]
 pub fn sinh(
-    /// The angle whose hyperbolic sine to calculate.
-    angle: AngleLike,
+    /// The hyperbolic angle whose hyperbolic sine to calculate.
+    value: Num,
```

Comment:
You can replace `Num` with `f64`. It will still allow integers. `Num` is only for places for integers and floats need separate code.

Reply:
Ah understood. I made the necessary changes.